pom.xml

```diff
@@ -1788,7 +1788,7 @@
                 <plugin>
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
-                    <version>1.4</version>
+                    <version>1.5</version>
```

Comment:
Even better upgrade it to the latest version - 1.11.2

Reply:
Yes. I updated it.